File: JavaProgram/program4.java
//Accept number from user and check whether number is even or odd


package LB.JavaProgram;

import java.util.Scanner;

public class program4 {
    public  boolean checkEven(int iNo)
    {   
        if(iNo%2==0)
        {
            return true;
        }
        else
        {
            return false;
        }
    }
    public static void main(String[] args) {
        Scanner s=new Scanner(System.in);

        int iValue=0;
        boolean bRet=false;

        System.out.println("Enter number to check wheater even or odd");
        iValue=s.nextInt();

        program4 p=new program4();

        bRet=p.checkEven(iValue);
       // System.out.println(bRet);

        if(bRet==true)
        {
            System.out.println("Even number");
        }
        else
        {
            System.out.println("Odd number");
        }      
    }
}

File: LBAssignment/MarvellousAssignment11/program4.c
//Accept number from user and display below pattern
//Input : 4
//output:    #        1       *      #        2       *      #        3       *      #        4       *

#include<stdio.h>

void Display(int iNo)
{
    int iCnt=0;
    for(iCnt=1;iCnt<=iNo;iCnt++)
    {
        printf("#\t %d\t *\t",iCnt);
    }
}
int main()
{
    int iValue=0;

    printf("Enter Value : ",&iValue);
    scanf("%d",&iValue);

    Display(iValue);
}